app/helpers/insured/group_selection_helper.rb

```diff
@@ -266,7 +266,8 @@ def is_member_checked?(benefit_type, is_health_coverage, is_dental_coverage, is_
     end
 
     def family_member_eligible_for_mdcr(family_member)
-      tmm = @family.households&.first&.tax_households&.map(&:tax_household_members)&.flatten
+      applicable_taxhouseholds = @family.households&.first&.tax_households&.select{|tx| TimeKeeper.date_of_record > tx[:effective_starting_on] && (tx[:effective_ending_on].nil? || (TimeKeeper.date_of_record < tx[:effective_ending_on]))}
```

Comment:
@polographer and I discussed rewriting this with a new scope on the tax household model.